package.json

```diff
@@ -223,14 +230,12 @@
     "promise-fs": "^2.1.1",
     "qr-manipulation": "git+https://github.com/brettz9/qr-manipulation.git",
     "query-result": "git+https://github.com/WebReflection/query-result.git",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
```

Comment:
While I know it can sometimes be faster to just change things and submit , I tend to have reasons for including things, so you might want to email me to ask why something that seems unecessary is in place before removing in case I don't catch it in my review, and then we have more back-and-forth after we later rediscover the need.

While we strictly speaking don't need these, we get a bunch of warnings with them on installation (which may distract from other more serious warnings that could occur) since some of our linting plugins have them as peer dependencies (I've sought changes in the responsible plugin so it won't be necessary, but still awaiting a reply).

Reply:
This branch is really in draft mode only. At this point, I'm not even sure that this is the correct way to refresh the build process. This is to explain that I will take every single of your comments before I consider the merge.